run-tests.sh

```diff
@@ -42,6 +42,16 @@ if ! $ANY_ARCH-w64-mingw32-gcc -E is-ucrt.c > /dev/null 2>&1; then
 fi
 rm -f is-ucrt.c
 
+echo "int main(){}" | $ANY_ARCH-w64-mingw32-clang -x c++ - -o has-cfguard-test.exe -mguard=cf
+if llvm-readobj --coff-load-config has-cfguard-test.exe | grep -q 'CF_INSTRUMENTED (0x100)'; then
```

Comment:
If I understand correctly - this check is only capable of discerning whether the compiler actually did handle `-mguard=cf` by setting the instrumentation flags on the main object file, while the rest of the toolchain might have been built without `--enable-cfguard`?

What happens with older versions of clang that don't recognize `-mguard=cf`, won't they error out when executing that clang command with an unrecognized option?

Reply:
The check should fail when mingw-w64-crt isn't built with `--enable-cfguard` because `_load_config_used` will be missing and then the CF flags won't exist.

You're right, older clang would error out on `-mguard=cf` so I will need to wrap this in another `if` block too.